portfolio/src/main/java/com/google/sps/Data/comment_object.java

```diff
@@ -0,0 +1,26 @@
+// Copyright 2019 Google LLC
```

Comment:
Why suddenly underscore separated class name?
Just Comment or CommentEntity is fine

Reply:
Just to make it clear. Actually had a lot of variable like comment.

Reply:
I am able to add comments.
To see comments , go to https://achoudhary-sps-summer20.df.r.appspot.com/comments

Reply:
[deployed link](https://achoudhary-sps-summer20.df.r.appspot.com/).
The comments are showing up now.
It seems I got something wrong earlier.